app/src/Controller/MessageController.php

```diff
@@ -105,35 +111,30 @@ public function index(Request $request, PaginatorInterface $paginator, ?string $
                 'Message.Actions.Delete' => 'delete',
             ];
             $subTitle = 'Entities.Message.untreated';
+            $messageStatus = null;
         }
-        $email = "";
 
-        $filterForm = $this->createForm(ActionsFilterType::class, null, ['avalaibleActions' => $messageActions, 'action' => $this->generateUrl('message_batch')]);
+        $filterForm = $this->createForm(ActionsFilterType::class, null, [
+            'avalaibleActions' => $messageActions,
+            'action' => $this->generateUrl('message_batch')
+        ]);
 
-        if ($type === MessageStatus::DELETED || $type === MessageStatus::VIRUS) {
+        if ($messageStatus && ($messageStatus->getId() === MessageStatus::DELETED || $messageStatus->getId() === MessageStatus::VIRUS)) {
             $filterForm->remove('actions');
         }
 
         /** @var User $user */
         $user = $this->getUser();
-        if ($user->getEmail() && in_array('ROLE_USER', $user->getRoles())) {
-            $email = stream_get_contents($user->getEmail(), -1, 0);
-        }
-        $em = $this->em;
-        $alias = $em->getRepository(User::class)->findBy(['originalUser' => $user->getId()]);
 
         $sortParams = [];
-        if ($request->query->has('sortDirection') && $request->query->has('sortField')) {
-            $sortParams['sort'] = $request->query->get('sortField');
-            $sortParams['direction'] = $request->query->get('sortDirection');
-        }
-        $searchKey = null;
-        if ($request->query->has('search')) {
-            $searchKey = trim($request->query->get('search'));
+        if ($request->query->has('sort') && $request->query->has('direction')) {
+            $sortParams['sort'] = $request->query->get('sort');
+            $sortParams['direction'] = $request->query->get('direction');
```

Comment:
Je n’aime pas trop utiliser `get` seul parce qu’on ne peut jamais être sûr du type de la valeur retournée

```suggestion
            $sortParams['sort'] = $request->query->getString('sort');
            $sortParams['direction'] = $request->query->getString('direction');
```